aviary/subsystems/geometry/geometry_builder.py

```diff
@@ -60,15 +60,16 @@ def build_pre_mission(self, aviary_inputs):
         geom_group = None
 
         if both_geom:
-            geom_group = CombinedGeometry(aviary_options=aviary_inputs,
-                                          code_origin_to_prioritize=code_origin_to_prioritize)
+            geom_group = CombinedGeometry(
```

Comment:
I wasn't able to remove aviary_inputs from the builders, or from groups that instantiate builders, because the builders sometimes need to make decisions based on those variables and settings. This is definitely true in our default builders for build_mission. I am less sure about build_pre_mission, and our internal builders might not need anything from aviary_options. However, we also have to support external builders where current users might already be using something from the provided aviary_inputs. 

It may be possible to refactor this more in the future to streamline the API and maybe get closer to removing aivary_inputs from the builders too.